terps/bocfel/io.cpp

```diff
@@ -35,6 +35,20 @@ extern "C" {
 #include "unicode.h"
 #include "util.h"
 
+#ifdef ZTERP_NO_STDIO
+#ifndef ZTERP_GLK_UNIX
+#error ZTERP_NO_STDIO requires a Unix Glk
+#endif
+
+#include <glkstart.h>
```

Comment:
Should this also be wrapped in `extern "C"`? I'll add the extern C wrapping to my remglk-rs library, but it couldn't be good to wrap this too, just in case, unless double wrapping is bad?

Reply:
Yep, nice catch.  I was building against (an experimental branch of) Gargoyle which sets `extern "C"` in the header, so I never noticed this.  I'll have it added in the next release.